roles/network/handlers/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Restart systemd-networkd
+  become: true
+  ansible.builtin.service:
+    name: systemd-networkd
+    state: restarted
+  when: network_allow_service_restart
```

Comment:
did you check whether a reload would be good enough? sounds like that could be less disruptive

Reply:
Indeed, a reload is sufficent. Changed to `reloaded`